tests/unittests/utils/test_qwen3_vl.py

```
@@ -398,5 +400,167 @@ def test_prepare_image_hwc_small_height(self):
+            def __exit__(self, *args):
+                pass
+
+        return _Mock()
+
+    def test_mode_none_video_dataset(self):
+        """mode=None on video dataset -> sample-level embeddings"""
+        model = self._make_mock_model()
+        ds = fo.Dataset()
+        ds.media_type = "video"
+        ds.add_sample(fo.Sample(
+            filepath="/tmp/test_video.mp4"
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_video.mp4"

(S108)